jvb/src/main/kotlin/org/jitsi/videobridge/relay/RelayMessageTransport.kt

```diff
@@ -147,7 +147,16 @@ class RelayMessageTransport(
             return null
         }
 
-        ep.addReceiver(relay.id, message.videoConstraints)
+        if (MultiStreamConfig.config.enabled) {
+            val sourceName = message.sourceName
+            if (ep.hasMediaSource(sourceName)) {
```

Comment:
Technically, the endpoint ID is not needed with source name in `AddReceiverMessage`, but it makes the lookup a little easier here. I'm not sure whether to keep both endpoint ID or to use only source name in the new mode? Keeping them both may create a false impression of an endpoint being another level of indirection, but source names are conference global (I think).

Reply:
I think it's cleaner to use only `sourceName`, but since you're reusing the message and have the endpoint ID you might as well use it. We can clean that up (remove endpointId) when we phase out the old code.

Reply:
yeah I think it's cleaner to use only `sourceName`